azure_pipelines/spm-framework.yml

```diff
@@ -304,7 +304,7 @@ jobs:
       targetType: 'inline'
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->